### PR TITLE
Feature/custom bounty duration

### PR DESCRIPTION
**Pull request by sbnetops**

### CODE REVIEW

File: src/polyswarm_api/endpoint.py [hunk 1]
```diff
@@ -269,38 +269,49 @@ def search_metadata(self, query):
             result_parser=resources.Metadata,
         )
 
-    def submit(self, artifact, artifact_name, artifact_type):
+    def submit(self, artifact, artifact_name, artifact_type, scan_config=None):
```
Comment (supernothing) [inline src/polyswarm_api/endpoint.py:272]: Could we create a sane URL default during this and make it the default based on the type? Would go a long way to making it less likely users encounter Steve's bug.

Comment (sbnetops) [reply src/polyswarm_api/endpoint.py:272]: Sure, what do you think it should be the default?

Comment (sbnetops) [reply src/polyswarm_api/endpoint.py:272]: Set to `more-time`

File: src/polyswarm_api/api.py [hunk 2]
```diff
@@ -114,13 +114,14 @@ def search_by_metadata(self, query):
...
 
-    def submit(self, artifact, artifact_type=resources.ArtifactType.FILE, artifact_name=None):
+    def submit(self, artifact, artifact_type=resources.ArtifactType.FILE, artifact_name=None, scan_config=None):
         """
         Submit artifacts to polyswarm and return UUIDs
 
         :param artifact: A file-like, path to file, url or LocalArtifact instance
         :param artifact_type: The ArtifactType or strings containing "file" or "url"
         :param artifact_name: An appropriate filename for the Artifact
+        :param scan_config: The scan configuration to be used
```
Comment (mrtizmoatwork) [inline src/polyswarm_api/api.py:124]: list the acceptable string values in this comment

**Pull request merged**